src/app/components/member/member-profile/member-profile.component.html

```diff
@@ -1,60 +1,86 @@
+                </li>
+              </ul>
+              <div class="tab-content ml-1" id="myTabContent">
+                <div class="tab-pane fade show active" id="basicInfo" role="tabpanel" aria-labelledby="basicInfo-tab">
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">ID</label>
+                    </div>
+                    <div class="col-md-8 col-6">{{member.id}} </div>
+                  </div>
+                  <hr />
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">First Name</label>
+                    </div>
+                    <div class="col-md-8 col-6">{{member.firstName}} </div>
+                  </div>
+                  <hr />
 
-<div class="row">
-<div class="col-sm-3 col-md-2 col-5">
-<label style="font-weight:bold;">Last Name</label>
-  </div>
-    <div class="col-md-8 col-6"> {{lastname}}</div>
-       </div>
-    <hr />
-    <div class="row">
-     <div class="col-sm-3 col-md-2 col-5">
-<label style="font-weight:bold;">Email</label>
-     </div>
- <div class="col-md-8 col-6"> {{email}} </div>
- </div>
-                                   
- </div>
- <div class="tab-pane fade" id="connectedServices" role="tabpanel" aria-labelledby="ConnectedServices-tab">
-   Facebook, Google, Twitter Account that are connected to this account
-  </div>
-                        
-</div>
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">Last Name</label>
+                    </div>
+                    <div class="col-md-8 col-6"> {{member.lastName}}</div>
+                  </div>
+                  <hr />
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">Gender</label>
+                    </div>
+                    <div class="col-md-8 col-6"> {{member.gender}} </div>
+                  </div>
+                  <hr />
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">DateOfBirth</label>
+                    </div>
+                    <div class="col-md-8 col-6"> {{member.dateOfBirth}} </div>
+                  </div>
+                  <hr />
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">Email</label>
+                    </div>
+                    <div class="col-md-8 col-6">{{member.email}} </div>
+                  </div>
+                  <hr />
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">DateCreated</label>
```

Comment:
We can show this as "Member Since" than just "DateCreated" which is a bit ambiguous for the reader.
Also if you see how this looks when you load it on the page, it shows a number that is not clear for ordinary user. So use pipe to convert that to readable month and year only. So it shows as, eg, February 2019.